### PR TITLE
Create institution request emails and reject institution request

### PR DESCRIPTION
**Pull request by MaianaB**
**Feature/Bug description:** The e-mail request for an institution registration and rejection of the request was in an old template, but the new template already existed when the request was accepted.

**Solution:** Changed the emails according to the design and created only one class sender for request and response (accepted or rejected), so removed the AcceptedInstitutionEmailSender and created RequestInstitutionEmailSender.

**TODO/FIXME:** n/a

**HOTMAIL**
req1uest_h
reject_inst_ht
rejec2

**GMAIL**

screenshot from 2018-05-23 11-27-29
screenshot from 2018-05-23 11-27-39

### CODE REVIEW

File: backend/models/request_institution.py [hunk 1]
```diff
@@ -4,9 +4,11 @@
 from . import Invite
 from . import Request
 from google.appengine.ext import ndb
+from custom_exceptions.fieldException import FieldException
```
Comment (Luiz-FS) [inline backend/models/request_institution.py]: Remove fieldException, this is unnecessary

File: backend/models/request_institution.py [hunk 2]
```diff
@@ -32,37 +34,46 @@ def create(data):
...
+            subject = get_subject('REJECTED_REQUEST_INSTITUTION')
+            html = 'rejected_institution_request_email.html'
 
-            sender_email = self.sender_key.get().email[0]
-            super(RequestInstitution, self).send_email(
-                host, sender_email, rejectMessage)
+        institution = self.institution_key.get()
+        institution_requested = self.institution_requested_key.urlsafe()
+        email_sender = RequestInstitutionEmailSender(**{
+            'receiver': self.sender_key.get().email[0],
```
Comment (raonismaneoto) [inline backend/models/request_institution.py]: extract this sender to a variable so you can reuse it in user_email

File: backend/util/strings_pt_br.py [hunk 3]
```diff
@@ -10,6 +10,9 @@
     'LINK_CONFIRM': 'Confirmação de vínculo',
     'REQUEST_USER': 'Solicitação de participação plataforma CIS',
     'RESPONSE_REQUEST_USER': 'Resposta à sua solicitação de participação da plataforma CIS',
+    'REQUEST_INSTITUTION': 'Solicitação de criação de instituição na plataforma CIS',
```
Comment (raonismaneoto) [inline backend/util/strings_pt_br.py]: Plataforma Virtual CIS*

File: backend/models/request_institution.py [hunk 4]
```diff
@@ -4,9 +4,11 @@
 from . import Invite
 from . import Request
 from google.appengine.ext import ndb
+from custom_exceptions.fieldException import FieldException
+from send_email_hierarchy import RequestInstitutionEmailSender
+from util.strings_pt_br import get_subject
```
Comment (Luiz-FS) [inline backend/models/request_institution.py]: Remove strings_pt_br, this is unnecessary

File: backend/models/request_institution.py [hunk 5]
```diff
@@ -4,9 +4,11 @@
...
 from . import get_deciis
 from custom_exceptions import FieldException
```
Comment (Luiz-FS) [inline backend/models/request_institution.py]: Duplicated import

**Pull request merged**